### ISSUE TITLE
Data Profiling Tool guidance document

### ISSUE THREAD

**Issue by sjurl** [ices-eg/DIG#475]

**Comment by sjurl** [comment]
Plan to be used as a basis for https://github.com/ices-eg/DIG/issues/528

**Comment by pcrjoana** [comment]
Draft guidelines have been written, and they will be circulated to the subgroup after review by @neil-ices-dk

**Comment by davidcurrie2001** [comment]
@pcrjoana is there a draft that can be circulated ahead of the DIG meeting in May?

**Comment by pcrjoana** [comment]
@davidcurrie2001 Yes! https://community.ices.dk/Committees/DIG/Master%20documents/Data-Profiling-Tool/Data%20profiling%20tool_guidelines.docx

**Comment by neil-ices-dk** [comment]
@pcrjoana thanks, i've made some further edits and comments,

**Comment by pcrjoana** [comment]
Thank you @neil-ices-dk. I'm looking at your comments right now, so the revised version will be available later today. :-)

**Comment by pcrjoana** [comment]
All changes accepted and @neil-ices-dk's suggestions have been implemented.

**Comment by davidcurrie2001** [comment]
@pcrjoana Is there any more work required on the DPT guidance document or can it be published?

**Comment by neil-ices-dk** [comment]
@pcrjoana can you accept all the changes and let me know for a final lookover before you send to pubs, thanks

**Comment by pcrjoana** [comment]
Thank you Neil! I'll finalise the guidelines this week.

**Comment by pcrjoana** [comment]
Guidelines published.
https://doi.org/10.17895/ices.pub.28204268.v1
